controllers/renovate_util.go

```diff
@@ -119,6 +151,7 @@ func (u ComponentDependenciesUpdater) GetUpdateTargetsBasicAuth(ctx context.Cont
 	targetsToUpdate := []updateTarget{}
 
 	for _, component := range componentList {
+		component := component
```

Comment:
While it works, it looks ugly. Can we have something like `component := comp`?

Reply:
ok